### DISCUSSION TITLE
No way to access Github, Support is silent. What should I do?

### DISCUSSION THREAD

**Discussion by inyutin-tr** [community/community#31070] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello! My name is Dmitry and recently I have encountered a very unpleasant problem: I can't access my account any more. 

My original account is https://github.com/inyutin. I am active user of Github: we use it for work projects and I maintain an open-source library [aiohttp_retry ](https://github.com/inyutin/aiohttp_retry). All my subproject are also stored at the Github. Moreover I often create issues or suggest improvements of different libraries too. Due to those reasons I access Github at least once a day. What I want to say here is that losing of my profile is a big loss for me.

At Thursday, 25th August, I found out that I can't log into Github account. The problem was that I can't pass through 2FA. Neither code from 2FA app or any one from recovery codes worked. I had a bad idea that it might be worth trying to recover the password. However, when I tried to recover it, I was also unable to go through 2fa too. And now my password seems to be invalidated, so the things have gotten worse. Github suggest to recover an account by ssh key. But I have no access to such a button due to password be invalid. You need to write a right password to see such an option.

So I tried to write an email to support. Three days have passed and I have no response. That's why I am trying to write things here. Github also recommends to do it. What should I do in such a situation? How should I contact Github to fix the problem?

P.S. I have a strong feeling that there is might be a bug on Github side. I don't have any arguments, of course, but I'll just give an idea. A few days before, I had changed the main email. More precisely, I deleted the main email from the email list and made another one the main one.  Perhaps this disabled my 2fa in some way.

**Comment by melvincwng** [comment, marked as answer]
@inyutin-tr, Hi there, for account related issues, members of the community won't be able to help you specifically.

However, I do have a suggestion. Please kindly contact [GitHub Support](https://support.github.com/contact/account) and raise a ticket explaining to them the situation whereby you are locked out of access in your main account.

Provide them a couple of days to investigate and resolve the issue.

Hope this clarifies and answers your query 😊

**Reply by queenofcorgis** [reply]
+1 to @melvincwng - please be patient with GitHub Support as they answer tickets based on their resourcing and in the order they were received.
